docs/source/sysadmin_guide/creating_genesis_block.rst

```diff
@@ -0,0 +1,131 @@
+   existing Sawtooth network.
+
+1. Ensure that the required user and validator keys exist:
+
+   .. code-block:: console
+
+      $ ls ~/.sawtooth/keys/
+      {yourname}.priv    {yourname}.pub
+
+      $ ls /etc/sawtooth/keys/
+      validator.priv   validator.pub
+
+   If these key files do not exist, create them as described in
+   :doc:`generating_keys`.
+
+#. Become the ``sawtooth`` user. In the following commands, the prompt
+   ``[sawtooth@system]`` shows the commands that must be executed as
+   ``sawtooth``.
+
+   .. code-block:: console
+
+      $ sudo -u sawtooth -s
+      [sawtooth@system]$ cd /tmp
+
+#. Create a batch to initialize the Settings transaction family in the genesis
+   block.
+
+   .. code-block:: console
+
+      [sawtooth@system]$ sawset genesis --key /etc/sawtooth/keys/validator.priv -o config-genesis.batch
+
+#. Create and submit a proposal to initialize the PoET consensus settings. This
+   command sets the consensus algorithm to PoET simulator, and then applies the
+   required settings.
+
+   .. code-block:: console
+
+      [sawtooth@system]$ sawset proposal create --key /etc/sawtooth/keys/validator.priv \
+      -o config.batch \
+      sawtooth.consensus.algorithm=poet \
+      sawtooth.poet.report_public_key_pem="$(cat /etc/sawtooth/simulator_rk_pub.pem)" \
+      sawtooth.poet.valid_enclave_measurements=$(poet enclave measurement) \
+      sawtooth.poet.valid_enclave_basenames=$(poet enclave basename)
+
+   This is a complicated command. Here's an explanation of the options:
+
+   ``--key /etc/sawtooth/keys/validator.priv``
+         Signs the proposal with this node's validator key. By default, only the
+         ``sawtooth`` user on this node can change on-chain settings. A later
+         step shows how you can submit a proposal that allows other users to
+         change on-chain settings.
+
+   ``sawtooth.consensus.algorithm=poet``
+         Changes the consensus algorithm to PoET.
+
+   ``sawtooth.poet.report_public_key_pem="$(cat /etc/sawtooth/simulator_rk_pub.pem)"``
+         Adds the public key for the PoET Validator Registry transaction
+         processor to use for the PoET simulator consensus.
+
+   ``sawtooth.poet.valid_enclave_measurements=$(poet enclave measurement)``
+         Adds a simulated enclave measurement to the blockchain. The
+         PoET Validator Registry transaction processor uses this value to check
+         signup information.
+
+   ``sawtooth.poet.valid_enclave_basenames=$(poet enclave basename)``
+         Adds a simulated enclave basename to the blockchain. The PoET
+         Validator Registry uses this value to check signup information.
+
+#. Create a batch to register the first Sawtooth node with the PoET Validator
+   Registry transaction processor. Without this command, the validator would not
+   be able to publish any blocks.
+
+   .. code-block:: console
+
+      [sawtooth@system]$ poet registration create --key /etc/sawtooth/keys/validator.priv -o poet.batch
```

Comment:
It is important that the key used here is the same key that the genesis validator uses. Might be worth noting.

Reply:
I'm concerned that putting this note here might add confusion (because it doesn't explain which keys don't HAVE to use the same genesis-node validator key). I talked to @aludvik and we agreed to put a note at the beginning of this procedure instead. My proposed wording is:

    "This procedure uses the same validator key for all commands that require a   
     key. In theory, some of these commands could use a different key, but        
     configuring multiple keys is more complicated. Describing how to use         
     multiple keys is outside the scope of this guide."